sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/plans/PlanTest.scala

```diff
@@ -61,7 +60,7 @@ trait CodegenInterpretedPlanTest extends PlanTest {
  * Provides helper methods for comparing plans, but without the overhead of
  * mandating a FunSuite.
  */
-trait PlanTestBase extends PredicateHelper with SQLHelper { self: Suite =>
+trait PlanTestBase extends PredicateHelper with SQLHelper with SparkTestSuite {
```

Comment:
super nit: seems nicer to `extends SparkTestSuite with PredicateHelper with SQLHelper`